### PR TITLE
supervised fworld comparison and defined metrics as well as changes to get jobczar to compile.

### PR DESCRIPTION
**Pull request by wumpusman**
added supervised, toggling for fworld to use heuristic or not, and changes so jobczar can run in golang.

### CODE REVIEW

File: go.work [hunk 1]
```diff
@@ -4,8 +4,8 @@ use ./emergent
 use ./axon
 use ./models
 use ./worlds
-use ./infra
-use ./infra/gengo/infra
+//use ./infra
```
Comment (qemqemqem) [inline go.work:7]: I thought this was being put in?

File: infra/go.mod [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module infra
+
+go 1.18
```
Comment (qemqemqem) [inline infra/go.mod:3]: What about the Requires?

**Review by qemqemqem — COMMENTED** [review]
Added comments

File: infra/go.mod (hunk 2 above)
Comment (wumpusman) [reply infra/go.mod:3]: It compiles for me, but fair enough. Will merge your version, and I'll merge my python code in a separate one.

**Pull request closed without merging**